libraries/standard/http/utest/http_utest.c

```diff
@@ -727,8 +765,10 @@ void test_Http_AddHeader_Single_Header_Insufficient_Memory()
 
     /* Run the method to test. */
     httpStatus = HTTPClient_AddHeader( &requestHeaders,
-                                       HTTP_TEST_HEADER_FIELD, HTTP_TEST_HEADER_FIELD_LEN,
-                                       HTTP_TEST_HEADER_VALUE, HTTP_TEST_HEADER_VALUE_LEN );
+                                       ( uint8_t * ) HTTP_TEST_HEADER_FIELD,
+                                       HTTP_TEST_HEADER_FIELD_LEN,
+                                       ( uint8_t * ) HTTP_TEST_HEADER_VALUE,
+                                       HTTP_TEST_HEADER_VALUE_LEN );
```

Comment:
Casts above are simply to remove warnings

Reply:
It is often better to cast in the #define at the top instead of casting everywhere it is being used. That said the need for casting usually indicates something is wrong, if this API takes strings then a const char* should suffice and using a standard string literal should not require a cast ...